website/src/components/Footer.tsx

```diff
@@ -96,22 +96,34 @@ export default class Footer extends React.Component<any, any> {
                                 <input type="checkbox" />
                                 <ul>
                                     <li>
-                                        <Link to="/#search-datacenter" onClick={this.codeSearchClicked}>
+                                        <Link
+                                            to="https://sourcegraph.com/start#search-datacenter"
```

Comment:
do these hashes actually take you anywhere?

Reply:
The will. The will open the modal – future enhance meant